README.md

```diff
@@ -94,9 +94,8 @@ This: [xdaforums.com](https://xdaforums.com/t/alldocube-t803-smile_1-bootloader-
 Or this: [subut](https://unisoc-android.github.io/subut/)
 
 ### Qualcomm 
-There's no Universal Qualcomm method, unfortunately.
-
-Although some of these might work for you:
+https://github.com/bkerler/edl May work depending on the type of qualcomm phone. May not work even with right loader from https://github.com/bkerler/Loaders
+if its a samsung or any scummy company which will lock firehose programmers to XBL bootloader version.
```

Comment:
This should probably be separated into an "EDL/Sahara" section or something, as it is not directly related to unlocking.

While some devices use EDL tricks to unlock, IIRC all modern devices store the bootloader bit in the secure section of the flash.